src/views/Correction.vue

```diff
@@ -852,6 +858,20 @@ export default class Correction extends Mixins(CommonMixin, DateMixin, EnumMixin
     }
   }
 
+  /** Array of valid components. Must match validFlags. */
+  readonly validComponents = [
+    'detail-comment-section',
+    'certify-form-section'
+  ]
+
+  /** Object of valid flags. Must match validComponents. */
+  get validFlags (): object {
+    return {
+      detailComment: this.detailCommentValid,
+      certifyForm: this.certifyFormValid
+    }
+  }
```

Comment:
Excellent use of the existing mechanism to implement auto scrolling in this component!
👍👍